source/integrations/setup-nginx-proxy-with-minio.rst

```diff
@@ -0,0 +1,193 @@
+            server minio-02.internal-domain.com;
+            server minio-03.internal-domain.com;
+            server minio-04.internal-domain.com;
+         }
+
+         server {
+            listen       80;
+            listen  [::]:80;
+            server_name  minio.example.net;
+
+            # To allow special characters in headers
+            ignore_invalid_headers off;
+            # Allow any size file to be uploaded.
+            # Set to a value such as 1000m; to restrict file size to a specific value
+            client_max_body_size 0;
+            # To disable buffering
+            proxy_buffering off;
+            proxy_request_buffering off;
+
+            location / {
+               proxy_set_header Host $http_host;
+               proxy_set_header X-Real-IP $remote_addr;
+               proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+               proxy_set_header X-Forwarded-Proto $scheme;
+
+               proxy_connect_timeout 300;
+               # Default is HTTP/1, keepalive is only enabled in HTTP/1.1
+               proxy_http_version 1.1;
+               proxy_set_header Connection "";
+               chunked_transfer_encoding off;
+
+               proxy_pass http://minio:9000/; # This uses the upstream directive definition to load balance
+            }
+         }
+
+         server {
+
+            listen       80;
+            listen  [::]:80;
+            server_name  console.example.net;
+
+            # To allow special characters in headers
+            ignore_invalid_headers off;
+            # Allow any size file to be uploaded.
+            # Set to a value such as 1000m; to restrict file size to a specific value
+            client_max_body_size 0;
+            # To disable buffering
+            proxy_buffering off;
+            proxy_request_buffering off;
+
+            location / {
+               proxy_set_header Host $http_host;
+               proxy_set_header X-Real-IP $remote_addr;
+               proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+               proxy_set_header X-Forwarded-Proto $scheme;
+               proxy_set_header X-NginX-Proxy true;
+
+               # This is necessary to pass the correct IP to be hashed
+               real_ip_header X-Real-IP;
+
+               proxy_connect_timeout 300;
+               
+               # To support websocket
+               proxy_http_version 1.1;
+               proxy_set_header Upgrade $http_upgrade;
+               proxy_set_header Connection "upgrade";
+               
+               chunked_transfer_encoding off;
+
+               proxy_pass http://minio:9001/; # This uses the upstream directive definition to load balance and assumes a static Console port of 9001
+            }
+         }
+
+The S3 API signature calculation algorithm does *not* support proxy schemes where you host either the MinIO Server API or Console GUI on a subpath, e.g. ``example.net/s3/`` or ``example.net/s3/console``.
```

Comment:
Can we drop the latinism `e.g.` and use the plain English `for example` or `that is`?

Reply:
`such as` could also work here to avoid repeating `example`.

Reply:
Done